example/sysjs.zig

```diff
@@ -5,8 +5,7 @@ const builtin = @import("builtin");
 pub const console = struct {
     /// console.log invokes the JS console.log API
     log: fn (string: []const u8) void,
-
-    log2: fn (string: []const u8, string2: []const u8) void,
+    log2: fn (string: []const u8, []const u8) void,
```

Comment:
isn't this illegal?

Reply:
oh i did it for testing but also no it's legal.